WooCommerce/Classes/ViewRelated/Dashboard/GoogleAds/GoogleAdsDashboardCardViewModel.swift

```diff
@@ -79,10 +85,27 @@ final class GoogleAdsDashboardCardViewModel: ObservableObject {
             DDLogError("⛔️ Error loading Google ads campaigns: \(error)")
         }
     }
+
+    func onViewAppear() {
+        viewAppeared = true
+    }
 }
 
 private extension GoogleAdsDashboardCardViewModel {
 
+    func trackEntryPointDisplayedIfNeeded() {
+        $canShowOnDashboard.removeDuplicates()
+            .combineLatest($syncingData.removeDuplicates(), $viewAppeared)
+            .filter { canShow, syncingData, viewAppeared in
+                return canShow && !syncingData && viewAppeared
```

Comment:
Super nit: Kindly consider adding a comment to explain this filter.

Reply:
Added comments in 7c4a27c9ca88a4183678c4c3053c0afea6e050d3.